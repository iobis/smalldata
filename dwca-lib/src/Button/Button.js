import React from 'react';

export default function Button() {
  return (
    <button onClick={() => console.log('hello')}>click me !!!!</button>
  )
}
