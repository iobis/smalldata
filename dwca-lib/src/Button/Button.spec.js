import Button from './Button'
import renderer from 'react-test-renderer'
import React from 'react'

describe('Button', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Button />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
