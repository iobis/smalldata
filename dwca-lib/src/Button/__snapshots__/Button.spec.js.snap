// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctrly 1`] = `
<button
  onClick={[Function]}
>
  click me
</button>
`;
